mycore-webtools/src/main/resources/components/webtools/config/mycore.properties

```diff
@@ -5,3 +5,6 @@
 
 MCR.Jersey.Resource.Packages=%MCR.Jersey.Resource.Packages%,org.mycore.webtools.session,org.mycore.webtools.upload
 MCR.URIResolver.xslIncludes.components=%MCR.URIResolver.xslIncludes.components%,sessionListing.xsl,processingGUI.xsl,msie-warning.xsl
+
+# filetypes that will be ignored for mainfile, if there are others
+MCR.Upload.IgnoreFiletypeForMainfile=".xml"
```

Comment:
The `"` in the property seems to be wrong

Reply:
Yes, thanks for the hint